test/uk/gov/hmrc/contactadvisors/service/EmailServiceSpec.scala

```diff
@@ -166,6 +167,6 @@ class EmailServiceSpec extends PlaySpec with MockitoSugar with ScalaFutures with
                                              |        "name": "Mr Joe Bloggs"
                                              |  }
                                              |}
-        """.stripMargin)
+        """.stripMargin).as[JsObject]
 
 }
```

Comment:
do we have the newly adder error cases covered in a unit test?

Reply:
there is no newly added error